### DISCUSSION TITLE
I get error when build with ios -

### DISCUSSION THREAD

**Discussion by Vietdq-dev** [codemagic-ci-cd/codemagic-docs#1800] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Open new build with Codemagic

### Expected results

I get this error below

<img width="1030" alt="Screenshot 2023-06-21 at 16 16 24" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/80184186/96711e19-3cc5-486a-91d6-e7b792d5deec">

### Actual results

Build success

### Build id (optional)

64919de380ad52f9f8cd6e1f

**Comment by himesh-cm** [comment, marked as answer]
Hi @Vietdq-dev ,
To resolve this issue, kindly visit https://appstoreconnect.apple.com/agreements/#/ and accept these agreements.
